components/BottomAppBar/examples/BottomAppBarTypicalUseExample.m

```diff
@@ -16,7 +16,7 @@
 
 #import "MaterialBottomAppBar+ColorThemer.h"
 #import "MaterialBottomAppBar.h"
-#import "MaterialButtons+ButtonThemer.h"
+#import "MaterialButtons+Theming.h"
```

Comment:
Please update the examples in a follow-up PR.

Reply:
Examples have been removed from this PR.